### ISSUE TITLE
enabling file logging as well

### ISSUE THREAD

**Issue by dfa1** [quarkiverse/quarkus-logging-json#190]
it is possible to enable logging to a file using this library? 

We tried "quarkus.log.file.json" but it not picking up this library at all. Any ideas? /cc @SlyngDK 
Now we have a workaround on our side, basically defining a custom processor:

``` java
  @BuildStep
  @Record(ExecutionTime.RUNTIME_INIT)
  LogFileFormatBuildItem setUpFormatter(Capabilities capabilities, LoggingJsonRecorder recorder, Config config) {
  ...
```

please note the use of `LogFileFormatBuildItem` rather than `LogConsoleFormatBuildItem` to enable usage of files.

**Comment by piyush-dig** [comment]
@SlyngDK I think a fix for this was merged in Quarkus via https://github.com/quarkusio/quarkus/pull/25505. 

Do we just need a new release or are there any changes required on the library side for this to work. I'll truly appreciate your help to resolve this.

**Comment by SlyngDK** [comment]
There needs to be added a `@BuildStep`, but also some work on the config structure. Maybe best to move some part of the config around, and create a new major version :thinking: 

I am not having the time to look at right now, I will take a look at it in Oct.

**Comment by dfa1** [comment]
@SlyngDK thank you :) I have some code already in my local fork: basically adding the `@BuildStep` but I stopped before started working in the config structure. Do you have any directions there?

**Comment by dfa1** [comment]
@SlyngDK hi! looks good to me... but I need a release (by company policy I can use only artifacts deployed on maven central).

Would it be possible to have a rc or beta? thanks!

**Comment by SlyngDK** [comment]
@gsmet Is it possible to create a RC release? Do you have any examples of doing that, not taking too much time.

**Comment by gsmet** [comment]
@SlyngDK not really, you can only push a release from the content of the repository.

**Comment by gsmet** [comment]
So to be clear: you can merge and release a CR by adjusting the `project.yml` to release a CR but you cannot release something from a PR.

**Comment by dfa1** [comment]
@SlyngDK @gsmet thank you! Since, there are some breaking changes, maybe at least one "2.0 release candidate" would be wise? I will happily provide early testing :)

**Comment by SlyngDK** [comment]
@dfa1 I have released 2.0.0.
